### DISCUSSION TITLE
Custom Domain issues with Github Pages and Google Domains

### DISCUSSION THREAD

**Discussion by callum-laing** [community/community#21964] [New to GitHub]
<p>Both <a href="http://www.callum-laing.com" rel="noopener nofollow ugc">www.callum-laing.com</a> and its alternate name are improperly configured</p>
<p>Domain’s DNS record could not be retrieved. For more information, see <a href="https://help.github.com/articles/setting-up-a-custom-domain-with-github-pages/" rel="noopener nofollow ugc">Learn more</a> (InvalidDNSError). We recommend you change this to a CNAME record pointing to <code>laing91.github.io</code> .</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/8/b/8bcd2b42ed9786e8eafc6241715a9b0f7739aeb9.png" data-download-href="/uploads/short-url/jWJWkV3vXKLzki4idPFw0m2AYat.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/86000107/181067920-01c5c489-6a9f-4ca9-801e-bf9119c4a826.png" alt="image" data-base62-sha1="jWJWkV3vXKLzki4idPFw0m2AYat" width="690" height="227" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/8/b/8bcd2b42ed9786e8eafc6241715a9b0f7739aeb9_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">796×263 19.4 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/9/0/90cddcddc0e55fad8d575953e2f481ff56487a18.png" data-download-href="/uploads/short-url/kEZOvAUDSKlbGqMH3tClUJL0Izu.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/86000107/181067921-05cd486a-0af4-4513-88b5-631120e2f61a.png" alt="image" data-base62-sha1="kEZOvAUDSKlbGqMH3tClUJL0Izu" width="690" height="190" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/9/0/90cddcddc0e55fad8d575953e2f481ff56487a18_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">704×194 10.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>I believe I’ve followed everything correctly, so I’m baffled, why isn’t it working…?</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>The host names you have configured have an extra <code>callum-laing.com</code> at the end. <code>www.callum-laing.com.callum-laing.com</code> is very different from <code>www.callum-laing.com</code>. The first version is resolvable right now, but I assume you want the second. :wink:</p>

**Comment by coutureone** [comment]
我想知道你解决了吗
